tree/ntuple/v7/src/RNTupleSerialize.cxx

```diff
@@ -40,8 +40,17 @@
 namespace {
 using RNTupleSerializer = ROOT::Experimental::Internal::RNTupleSerializer;
 
-std::uint32_t SerializeField(const ROOT::Experimental::RFieldDescriptor &fieldDesc, ROOT::DescriptorId_t onDiskParentId,
-                             ROOT::DescriptorId_t onDiskProjectionSourceId, void *buffer)
+#define GLUE_(a, b) a##b
```

Comment:
This is a duplicate of existing macros in `RConfig.cxx` (eg. `_R__JOIN_`)

Reply:
I suspected we had one but I couldn't find it, thanks